### DISCUSSION TITLE
Making toolbar global

### DISCUSSION THREAD

**Discussion by CodeDoctorDE** [LinwoodDev/Butterfly#627] [Polls]
Hello,
Tools and configuring the toolbar is an essential customization feature in butterfly.
But currently people are confused about it.
I'm thinking about making it global to make it easier for users to change it and keep it for all documents you want to edit.

An alternative to this (option 3):
We could add both and a selector to the toolbar to toggle between these. In this change we could add an option to have multiple toolbars and toggle between these.

Some things to consider:
- What about dark/light mode and white/black pens, how should we handle these?

Related to https://github.com/LinwoodDev/Butterfly/issues/613

**Implemented as Pack Toolbars in 2.5**

**Comment by yotama9** [comment]
The ability to configure the toolbar the way I need is appealing to me. If I think of a student, the features he needs in a math class, for example, are different from when he's playing D&D. But, on the other hand, if I'm realistic, most people will have 2-3 setups at most. Allowing the switch seems like the best option to me.

**Reply by CodeDoctorDE** [reply]
Thanks for your opinion.
With the 3rd option you could keep your current toolbar if you want and make it global.
What do you mean with switch?

**Comment by MrGumpIT** [comment]
The idea of ​​having different bars is good, however I tend to create it once and it almost works for everything.  It would be useful to be able to make it the default on each project but still maintain the use of templates
